maven/features/org.jboss.tools.maven.feature/pom.xml

```diff
@@ -5,10 +5,9 @@
 	<parent>
 		<groupId>org.jboss.tools.maven</groupId>
 		<artifactId>features</artifactId>
-		<version>1.9.100-SNAPSHOT</version>
+		<version>1.9.300-SNAPSHOT</version>
 	</parent>
 	<groupId>org.jboss.tools.maven.features</groupId>
 	<artifactId>org.jboss.tools.maven.feature</artifactId>
 	<packaging>eclipse-feature</packaging>
-	<version>1.9.200-SNAPSHOT</version>
-</project>
+	</project>
```

Comment:
why remove version here?

Reply:
Because previous bump was done on a single bundle instead of a whole tree